### DISCUSSION TITLE
translation?

### DISCUSSION THREAD

**Discussion by kuze7ay** [Steve-xmh/scl#93] [General]
The best... truly the best.Will it be translated to english soon? OR is there a way to change language?

**Comment by Steve-xmh** [comment]
Sorry about that, I'm very busy at working for other project and things in real life so there's only Chinese language. I mainly present this launcher and the framework to the Players from China. Including the documents. And honestly those document and text are written for myself. 😢

**Reply by kuze7ay** [reply]
NO problem. It can still work. but I ought to say , This project is pure magnificent, has better UI then all other(including the official).

**Comment by Steve-xmh** [comment]
If there is a large amount of people outside of China like my launcher then I maybe can do for an English translation for SCL.
